### ISSUE TITLE
Overal styling

### ISSUE THREAD

**Issue by Zaturrby** [augmenting-masterpieces/AugmentM#6]
Finetuning the overall styling.

- Do some block need a bit of air? 
- Are all headers formatted the same way? (white ones especially)
- Subtitle styling? 
- Masonry styling?

**Comment by Zaturrby** [comment]
So I did a lot of little styling things! The commit's show which ones. What should still be done?

1.  I think there are some oddities. Like the use of the accent-color, or the use of the little black borders at the bottom, also there is the underline link styling in the footer. It seems to me that all three either need to be used elsewhere in the site, or be gone. 
2. There is the long block beneath analyse and ideate. Im still doubting wether to split it up like I did with the video's, or to do something else with it. The video treatment might be best though.
3. I believe there was a logo? Title styling in the video's is done quite differently. Need to have the stuffs for that, if we want to be exactly the same.
